cmake/linux/CMakeLists.txt

```diff
@@ -2,3 +2,16 @@ INSTALL(FILES lmms.png                  DESTINATION "${DATA_DIR}/pixmaps")
 INSTALL(FILES lmms                      DESTINATION "${DATA_DIR}/menu")
 INSTALL(FILES lmms.desktop              DESTINATION "${DATA_DIR}/applications")
 INSTALL(FILES lmms.xml                  DESTINATION "${DATA_DIR}/mime/packages")
+
+SET(EXEC "env QT_X11_NO_NATIVE_MENUBAR=1 lmms")
```

Comment:
lmms is the menu file.... though a recent e-mail on the debian list suggests that soon this will go away since using a desktop file is much more common.
I can change the variables if you'd rather I did